devtools/common/src/main/java/io/quarkus/cli/commands/writer/Writer.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * 
+ */
+package io.quarkus.cli.commands.writer;
+
+import java.io.IOException;
+
+/**
+ * @author guillaumedufour
+ *         This writer provide a way to write direct a file or inside a zip for project creation
+ *
+ */
+public interface Writer {
```

Comment:
I'm not sure about the name of this... What concerns me is that it can be easily confused with `java.io.Writer` when reading the code. WDYT?

Reply:
yes i am totally agree but don't find a better name until now.
It can be CommandWriter or TemplateWriter, or StarterWriter or any other idea ?

Reply:
`ProjectWriter` maybe?